db/model/botData.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/botData.js
+ *
+ * Bot webapps will have data that refocus will want to retrieve
+ * or store. This table will define the data information to be
+ * used manipulate the bots data.
+ */
+
+const constants = require('../constants');
+
+const assoc = {};
+
+module.exports = function user(seq, dataTypes) {
```

Comment:
function name - user - is incorrect.
Side note - I don't think we need to name these functions at all - anonymous would do. However, other models seems to assign name to functions